scripts/covid_hosp_prod.R

```diff
@@ -76,13 +90,15 @@ forecaster_fns <- list2(
     fcst
   }
 )
-indices <- seq_along(forecaster_fns)
+forecaster_fn_names_ <- names(forecaster_fns)
 
-rlang::list2(
+
+# ================================ PARAMETERS AND DATE TARGETS ================================
+parameters_and_date_targets <- rlang::list2(
   tar_target(aheads, command = -1:3),
-  tar_target(forecasters, command = indices),
-  tar_target(
-    name = nhsn_latest_data,
+  tar_change(
+    nhsn_latest_data,
+    change = get_socrata_updated_at("https://data.cdc.gov/api/views/mpgq-jmmr"),
```

Comment:
We're going to have to watch this pretty carefully, I was getting some weird behavior. I hope it works

Reply:
Fair, I haven't really beta tested this. I haven't seen it _not run_ yet, which is the safer fail case.